napari/utils/info.py

```diff
@@ -0,0 +1,71 @@
+import platform
+
+import sys
+
+import napari
+
+
+def sys_info(as_html=False):
```

Comment:
is this just being left in here for comparison purposes and will get deleted if we go with the `napari_info` approach?

Reply:
ah, that goes if we go with `napari_info`

Reply:
yup, makes sense